stats/stats.go

```diff
@@ -197,6 +197,23 @@ func (st *SampleTags) IsEqual(other *SampleTags) bool {
 	return true
 }
 
+func (st *SampleTags) Contains(other *SampleTags) bool {
+	if st == other {
+		return true
+	}
+	if st == nil || other == nil || len(st.tags) < len(other.tags) {
```

Comment:
Ah, actually, I'm not sure, but I think that if `other == nil || len(other.tags) == 0`, then we should always return `true`, since every set contains the empty set.

Reply:
I have no idea what ruby does, but in the case of `SampleTags`, a `nil` pointer is basically the same as an empty tag set. I'm not very familiar with how tagging of submetrics is supposed to work and if this is the desired behaviour though, just that mathematically the empty set is a subset of all sets.

Reply:
I don't really care about that. Let's make that return `true` then.